src/SortingAlgorithms.jl

```diff
@@ -1,6 +1,7 @@
 module SortingAlgorithms
 
 using Compat
+import Compat.view
```

Comment:
put a minimum version on Compat in REQUIRE. you shouldn't need me to tell you this again.